includes/classes/Feature/WooCommerce/WooCommerce.php

```diff
@@ -648,4 +649,19 @@ public function maybe_disable_decaying( $is_decaying_enabled, $settings, $args )
 		_deprecated_function( __METHOD__, '4.7.0', "\ElasticPress\Features::factory()->get_registered_feature( 'woocommerce' )->products->maybe_disable_decaying()" );
 		return $this->products->maybe_disable_decaying( $is_decaying_enabled, $settings, $args );
 	}
+
+	/**
+	 * Add taxonomies that should be woocommerce attributes.
+	 *
+	 * @param array $attribute_taxonomies  Attribute taxonomies.
+	 * @return array $attribute_taxonomies Attribute taxonomies.
+	 */
+	public function add_taxonomy_attributes( $attribute_taxonomies ) {
```

Comment:
```suggestion
	public function add_taxonomy_attributes( array $attribute_taxonomies ) : array {
```
Especially for methods we have more control on, we are trying to specify types, @oscarssanchez :)

Reply:
Changed here @felipeelia